tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/NewApprenticeshipProvider/HidePassedNotificationTests.cs

```diff
@@ -0,0 +1,467 @@
+                apprenticeshipIds: new[] { apprenticeshipId });
+
+            await User.AsProviderUser(providerId, ProviderType.Apprenticeships);
+
+            await TestData.UpdateApprenticeshipQASubmission(
+                submissionId,
+                providerAssessmentPassed: null,
+                apprenticeshipAssessmentsPassed: null,
+                passed: null,
+                lastAssessedByUserId: User.UserId.ToString(),
+                lastAssessedOn: Clock.UtcNow);
+
+            //create provider assessment
+            Clock.UtcNow = Clock.UtcNow.AddDays(1);
+            var PassedQAOn = Clock.UtcNow;
+            await TestData.CreateApprenticeshipQAProviderAssessment(
+                submissionId,
+                adminUserId,
+                Clock.UtcNow,
+                true,
+                null,   //Compliance Comments
+                ApprenticeshipQAProviderComplianceFailedReasons.None,
+                true, //Style passed,
+                null,   //Style Comments
+                ApprenticeshipQAProviderStyleFailedReasons.None
+                );
+
+            // Act
+            var response = await HttpClient.PostAsync(
+                $"new-apprenticeship-provider/hide-passed-notification?providerId={providerId}&returnUrl=/", null);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+        }
+
+
+        [Fact]
+        public async Task Post_InProgressQAReturnsRedirect()
+        {
+            // Arrange
+            var ukprn = 12345;
+            var email = "somebody@provider1.com";
+            var providerName = "Provider 1";
+            var providerUserId = $"{ukprn}-user";
+            var adminUserId = $"admin-user";
+            Clock.UtcNow = new DateTime(2019, 5, 17, 9, 3, 27, DateTimeKind.Utc);
+            var requestedOn = Clock.UtcNow;
+
+            var providerId = await TestData.CreateProvider(
+                ukprn: ukprn,
+                providerName: providerName,
+                apprenticeshipQAStatus: ApprenticeshipQAStatus.InProgress,
+                providerType: ProviderType.Apprenticeships);
+
+            await TestData.CreateUser(providerUserId, email, "Provider 1", "Person", providerId);
+            await TestData.CreateUser(adminUserId, "admin", "admin", "admin", null);
+            var apprenticeshipId = await TestData.CreateApprenticeship(ukprn);
+
+            //create submission
+            var submissionId = await TestData.CreateApprenticeshipQASubmission(
+                providerId,
+                submittedOn: Clock.UtcNow,
+                submittedByUserId: providerUserId,
+                providerMarketingInformation: "The overview",
+                apprenticeshipIds: new[] { apprenticeshipId });
+
+            //update qa submission
+            await TestData.UpdateApprenticeshipQASubmission(
+                submissionId,
+                providerAssessmentPassed: null,
```

Comment:
One of these two should be non-null and there should be a corresponding provider/apprenticeship assessment

Reply:
added a TestData.CreatedApprenticeshipQAProviderAssessment now.